src/markdown_generators.js

```diff
@@ -205,6 +198,27 @@ const generateCourseHomeMarkdown = (courseData, pathLookup) => {
   }
 }
 
+const generateCourseHomePdfMarkdown = (courseData, pathLookup) => {
+  /**
+   * Generate markdown files representing PDF viewer pages for PDF's mentioned on the course home page
+   */
+
+  return courseData["course_files"]
+    .filter(
+      file =>
+        file["file_type"] === "application/pdf" &&
+        file["parent_uid"] === courseData["uid"]
+    )
+    .map(file => {
+      const { path: parentPath } = pathLookup.byUid[file["parent_uid"]]
+      return {
+        name: `${path.join(parentPath, helpers.stripPdfSuffix(file["id"]))}.md`,
+        data: generatePdfMarkdown(file, courseData)
+      }
+    })
+    .filter(Boolean)
```

Comment:
I thought you could filter out null or empty elements this way because `Boolean(x)` is a truthy expression?  Should it me something like `.filter(file => file)` instead?

Reply:
`Boolean(x)` works but `{}` is not a falsey value in javascript and all items will be something like `{ name, data }` anyway

Reply:
Right, yea.  I removed the filter as on second thought there shouldn't be any need for it.